packages/kit/assets/components/error.svelte

```diff
@@ -1,3 +1,11 @@
+<script context="module">
+	export async function load({ error, status }) {
```

Comment:
There's not really a reason for this to be async, but I don't feel particularly strongly about it.